src/ui/marker.js

```diff
@@ -33,6 +34,7 @@ type Options = {
  *   Options are `'center'`, `'top'`, `'bottom'`, `'left'`, `'right'`, `'top-left'`, `'top-right'`, `'bottom-left'`, and `'bottom-right'`.
  * @param {PointLike} [options.offset] The offset in pixels as a {@link PointLike} object to apply relative to the element's center. Negatives indicate left and up.
  * @param {string} [options.color='#3FB1CE'] The color to use for the default marker if options.element is not provided. The default is light blue.
+ * @param {number} [options.scale=1] The scale to use for the default marker if options.element is not provided. The default is 1.
```

Comment:
"The default is 1." Minor nit pick, that's okay, but for the most part in the rest of the docs the default value is omitted from the description and left to documentation.js to format the default from the JSDoc.

I know I put the default in the color option above, though since not everyone speaks hex codes it's probably justified in the color option.

Reply:
Great point, I'll remove that sentence and leave it up to documentation.js to format the default.